shap/plots/_beeswarm.py

```diff
@@ -441,6 +441,7 @@ def summary_legacy(shap_values, features=None, feature_names=None, max_display=N
                  class_inds=None,
                  color_bar_label=labels["FEATURE_VALUE"],
                  cmap=colors.red_blue,
+                 print_values_legend_multioutput=False,
```

Comment:
May I suggest to rename this as `show_values_in_legend`? I think we can leave `multioutput` out of the param name.

Then we can make clear it's for multioutput only in the docstring.

Reply:
Done :)